docs/deployment/pull-backup.rst

```diff
@@ -301,3 +301,101 @@ a backup may be the following command::
+  borgs@borg-server$ chmod go-w ~/.ssh/authorized_keys
+
+``install -m 700 -d ~/.ssh/``
+
+  Create directory ~/.ssh with correct permissions if not exists yet.
+
+``ssh-keygen -N '' -t rsa  -f ~/.ssh/borg-client_rsa``
+
+  Create RSA key dedicated to communication with borg-client.
+
+``{ echo -n 'command="borg serve --append-only --restrict-to-path ~/repo",restrict '; cat ~/.ssh/borg-client_rsa.pub; } >> ~/.ssh/authorized_keys``
+
+  Add borg-client's public key record to ~/.ssh/authorized_keys with force command and restrictions.
+  Client restricted to use specified repository path and to use only append operations.
+  The commands like *prune* and *compact* have to be executed another way, for example directly on *borg-server* side.
+
+``chmod go-w ~/.ssh/authorized_keys``
+
+  Fix permissions of ~/.ssh/authorized_keys.
+
+
+Pull operation
+~~~~~~~~~~~~~~~~~~~~~~
+
+Execute borg command (init repo in this example) on *borg-server*::
+
+  borgs@borg-server$ (
+    eval $(ssh-agent) > /dev/null
+    ssh-add -q ~/.ssh/borg-client_rsa
+    echo 'complicated & long' | \
+      ssh -A -o StrictHostKeyChecking=no borgc@borg-client "BORG_PASSPHRASE=\$(cat) borg --rsh 'ssh -o StrictHostKeyChecking=no' init --encryption repokey ssh://borgs@borg-server/~/repo"
+    kill "${SSH_AGENT_PID}"
+  )
+
+Parentheses around commands are needed to exclude interferention with possibly already running ssh-agent.
+Parentheses not needed in case of using dedicated bash process.
+
+``eval $(ssh-agent) > /dev/null``
+
+  Run SSH agent in background and export related environment variables to current bash session.
+
+``ssh-add -q ~/.ssh/borg-client_rsa``
+
+  Load SSH private key dedicated to communication with borg-client to SSH agent.
+  Look at ``man 1 ssh-add`` for more detailed explanation.
+
+  Care needs to be taken when loading key to SSH agent. Users on the *borg-client* having read/write permissions to
+  agent's UNIX-domain socket (at least borgc and root in our case) can access the agent on *borg-server* through the
+  forwarded connection and use loaded keys for authenticate using the identities loaded into the agent
+  (look at ``man 1 ssh`` for more detailed explanation). Therefore there are some security considerations:
+
+  * Private key loaded to agent must not be used to access anywhere else.
+  * The keys meant to be loaded to agent must be specified explicitly, not from default locations.
+  * The *borg-client*'s public key record at *borgs@borg-server:~/.ssh/authorized_keys* must be as restrictive as possible.
+
+``echo 'complicated & long' | ssh -A -o StrictHostKeyChecking=no borgc@borg-client "BORG_PASSPHRASE=\$(cat) borg --rsh 'ssh -o StrictHostKeyChecking=no' init --encryption repokey ssh://borgs@borg-server/~/repo"``
+
+  Issue *borg init* command to be executed at *borg-client*.
+
+  *'complicated & long'* is the password used for encrypt the key of new repository. The password passed via stdin,
```

Comment:
if you use `your secure borg key passphrase`, you do not need to explain what it is here.

Reply:
Fixed.